### ISSUE TITLE
Create on Realm button hard to read

### ISSUE THREAD

**Issue by Gengar-lab** [Offroaders123/Dark-Mode#3]
![Screenshot_Minecraft](https://user-images.githubusercontent.com/42458260/141333874-ea1fb812-4bd1-47c1-bc8b-550390663bac.jpg)

Also i suggest to change the light pink colour of the button to add more contrast with respect to dark background.

**Comment by PiSaucer** [comment]
Mojang recently changed the color of realms everywhere to this new purple color so it's not hard to read anymore:
![image](https://github.com/Offroaders123/Dark-Mode/assets/33438654/92adf59b-5594-42e3-a3be-97105fc6783f)

**Comment by Offroaders123** [comment]
Glad they made it nicer to look at in general! It looks better with Dark Mode now too. This is probably the color I would have made if I ended up fixing it on my own. I don't have much experience with making the darker versions of button colors for saturated ones. Usually I use Multiply with a `#494949` layer on top of the original texture, and that adds the dark theme to the base texture. I think I tried doing that with the original pink button, but it didn't look right.
